### ISSUE TITLE
Turrets should need pressure to run

### ISSUE THREAD

**Issue by Shibva** [TeamPneumatic/pnc-repressurized#1026]
### Describe the feature

Ive noticed that the turret itself works on its own with no power source; it should require bars of pressure to run

### Reasons why it should be considered

not only does it seem a bit OP, but it's also illogical at the same time. the Machine Gun item needed to make it is also a weapon that can be used yet to use it, you would need to have pressure/Bar for it to fire. With that logic, it would make sense that the turret would need the pressure to run, yet it doesn't. Where does it get the air that is needed for it to fire the component that needs pressure to use? out of thin air? there's no compressor built into it. 

It doesn't make sense, not only that its a bit broken because what to stop someone from just placing one down and using it to make the specific tasks easier; maybe if it had an air tank attached to it, but it doesn't make any sense for something that needs pressure to run to have a sentry that doesn't need it at all to run 

In conclusion; it would only make sense for it to require bars/pressure for it to run and with that said it would also make sense for capacity upgrades to be added to allow for a "Reserve" of some sort 

### Additional details

upgrades can be added to it as well if this is considered, and in general upgrades or not; it would be recommended to bring some balance to the turret

**Comment by desht** [comment]
I don't plan to add pressure requirements to Sentry Turrets, no.  I fully get that it's inconsistent, but it's not OP, since ammo still needs to be supplied, the turret can't move, it's subject to line of sight restrictions, and the damage isn't huge.  Making them require pressure would turn them from OK to "why bother".

I have considered an Advanced Sentry Turret which does require pressure, but would be far more upgradeable, taking the upgrades that the held Minigun takes (Speed/Item Life/Volume etc.).  That might happen at some point if I have time to ever implement it...
